projects/examples/src/components/datagrid/datagrid-detail-row.example.component.ts

```diff
@@ -14,10 +14,27 @@ interface Data {
  * Shows an expandable detail template within the rows.
  */
 @Component({
-    selector: 'vcd-datagrid-show-hide-example',
+    selector: 'vcd-datagrid-detail-row-example',
     template: `
-        <vcd-datagrid [gridData]="gridData" (gridRefresh)="refresh($event)" [columns]="columns">
-            <ng-template let-record="record"> DETAILS: {{ record.value }} </ng-template>
+        <div>
```

Comment:
A `div` should be used for containers that do layout. If it's just a paragraph, use `<p>`